applications/openshift/api-server/api_server_api_priority_gate_enabled/tests/ocp4/e2e.yaml

```diff
@@ -0,0 +1,2 @@
+---
+default_result: NOT-APPLICABLE
```

Comment:
Some of these aren't lining up with the results from CI.

Reply:
might be related to https://github.com/ComplianceAsCode/content/pull/11760

Reply:
but e2e here should catch the error tho

Reply:
Rebased to pickup the fix you landed for platforms.